app/components/createContainer/createContainerController.js

```diff
@@ -233,6 +234,156 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     return config;
   }
 
+  function confirmCreateContainer(cb) {
+    Container.query({
+      all: 1,
+      filters: {name: [$scope.config.name]}
+    }, function (data) {
+      var confirmDialog = false;
+      var container;
+      // Prompt if we found name to confirm replacement
+      for (var c in data) {
+        for (var n in data[c].Names) {
+          if (data[c].Names[n] === '/' + $scope.config.name) {
+            confirmDialog = true;
+            container = data[c];
+          }
+        }
+      }
+      if (confirmDialog) {
+        ModalService.confirmDeletion(
+          'A container with the same name is already present on this host. Do you want to remove it?',
+          function onConfirm(confirmed) {
+            if(!confirmed) { return cb(false); }
+            else {
+              // Remove old container
+              ContainerService.remove(container, true)
+              .then(function success(data) {
+                Notifications.success('Container Removed', container.Id);
+                return cb(true);
+              })
+              .catch(function error(err) {
+                Notifications.error('Failure', err, 'Unable to remove container');
+                return cb(false);
+              });
+            }
+          }
+        );
+      } else {
+        return cb(true);
+      }
+    }, function error(err) {
+      Notifications.error('Failure', err, 'Unable to retrieve containers');
+      return cb(false);
+    });
+  }
+
+  function loadFromContainerSpec() {
+    // Get container
+    Container.get({id: $stateParams.from}, function(d) {
+      $scope.fromContainer = new ContainerViewModel(d);
+      // Add Config
+      $scope.config = d.Config;
```

Comment:
Yeah that's right, I can re-use configFromContainer for a part, but need to transform many values to match with the view